.github/workflows/e2e-tf-deployment.yaml

```diff
@@ -9,10 +9,13 @@ on:
       - 'src/main/charts/bitbucket/**'
       - 'src/main/charts/confluence/**'
       - 'src/main/charts/jira/**'
+  pull_request:
```

Comment:
this code change is not addressing the requirement in [CLIP-1548](https://bulldog.internal.atlassian.com/browse/CLIP-1548)